CorsixTH/CMakeLists.txt

```diff
@@ -185,6 +198,14 @@ if(MSVC AND CORSIX_TH_USE_VLD)
   endif()
 endif()
 
+# Launch script to facilitate out of source builds on Linux
+if(UNIX)
+  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/run-corsixth-dev.sh.in ${CMAKE_CURRENT_BINARY_DIR}/run-corsixth-dev.sh @ONLY)
```

Comment:
The file is run-corsix-th-dev.sh.in with a dash in corsix-th.

Reply:
Ah, thanks. I changed the file name in a later commit and didn't fix this line.